### PR TITLE
Make commands executable on multiple OS + ArchLinux commands

### PR DESCRIPTION
**Pull request by martinfrancois**
Tests are still missing but may be added later...

### CODE REVIEW

File: src/ch/imedias/rsccfx/model/CommandHandler.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        new Command("-compresslevel", "-CompressLevel", "-CompressionLevel", null);
+    vncViewerQuality =
+        new Command("-quality", "-QualityLevel", "-QualityLevel", null);
+    vncViewerBgr233 =
+        new Command("-bgr233", "-LowColorLevel", "-LowColorLevel", null);
+
+  }
+
+  private String determineOs() {
+    if (System.getProperty("os.version").contains("ARCH")) {
```
Comment (rmosimann) [inline src/ch/imedias/rsccfx/model/CommandHandler.java:63]: Is it really os.version, not os.name, like for the others

File: src/ch/imedias/rsccfx/model/CommandHandler.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  /**
+   * Sets the currently used OS and also initializes all commands.
+   */
+  public CommandHandler() {
+    String os = determineOs();
+    setOs(os);
+    LOGGER.info("Running on the OS: " + os);
+    initializeCommands();
+  }
+
```
Comment (rmosimann) [inline src/ch/imedias/rsccfx/model/CommandHandler.java:38]: Please add descriptive comment

**Review by rmosimann — COMMENTED** [review]
Please review the two comments and then merge

File: src/ch/imedias/rsccfx/model/CommandHandler.java (hunk 1 above)
Comment (martinfrancois) [reply src/ch/imedias/rsccfx/model/CommandHandler.java:63]: no, that's correct!

**Pull request merged**